x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/collapsed_item_actions.test.tsx

```diff
@@ -149,6 +153,7 @@ describe('CollapsedItemActions', () => {
     expect(wrapper.find(`[data-test-subj="deleteRule"] button`).prop('disabled')).toBeFalsy();
     expect(wrapper.find(`[data-test-subj="deleteRule"] button`).text()).toEqual('Delete rule');
     expect(wrapper.find(`[data-test-subj="updateApiKey"] button`).text()).toEqual('Update API key');
+    expect(wrapper.find(`[data-test-subj="runRule"] button`).text()).toEqual('Run rule');
   });
 
```

Comment:
Can we test that runSoon is called on button click too?

Reply:
Yes, I can add that!

Reply:
Resolved in https://github.com/elastic/kibana/pull/139848/commits/6299f9396aac9ded30dc345cde01abddfdf13ae6